network/retrieve/wire.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 The Swarm Authors
```

Comment:
could we call this file `messages`?

Reply:
it is called `wire` because of `wire protocol`. if you demand i rename it i will rename it but i see no real reason to do so.